config/targets.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-env node */
+
+module.exports = {
+  browsers: [
```

Comment:
it's what `ember init` gives you but beyond that I don't know since we aren't actually using this in any of our apps